js/form.js

```diff
@@ -13,3 +16,29 @@ form.addEventListener('submit', (event) => {
 
 document.getElementById('error-text').style.color = 'Red';
 document.getElementById('error-text').style.fontSize = '120%';
+
+const savedInfo = JSON.parse(localStorage.getItem('myFormInfo'));
+if (savedInfo) {
+  fullName.value = savedInfo.name;
+  email.value = savedInfo.email;
+}
+
+function saveFormInfo() {
+  const myFormInfo = {
+    name: fullName.value,
+    email: email.value,
+  };
+  localStorage.setItem('myFormInfo', JSON.stringify(myFormInfo));
+}
+
+if (localStorage.getItem('textareaData')) {
+  textarea.value = localStorage.getItem('textareaData');
+}
+
+textarea.addEventListener('input', () => {
+  const data = textarea.value;
+  localStorage.setItem('textareaData', data);
+});
```

Comment:
- [ ] It's a requirement that the form data for all inputs should be stored in a single object, and saved to a single local storage registry.  In your code, you store the name and the email in an object and the `TeaxtArea` data is stored separately, kindly fix this by storing the message in the object holding the name and email as well.
e.g
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```

Project requirement screenshot (check the last point below)
![image](https://github.com/Chimoneg27/Personal-Portfolio/assets/81932478/c118e001-1aa9-47f7-ab56-a692af6dce35)